mteb/tasks/Retrieval/eng/MLQuestions.py

```diff
@@ -0,0 +1,100 @@
+import csv
+
+from huggingface_hub import snapshot_download
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class MLQuestionsRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="MLQuestions",
+        dataset={
+            "path": "McGill-NLP/mlquestions",
+            "revision": "83b690cb666c5a8869e7f213a877bbd24a642d7c",
+        },
+        reference="https://github.com/McGill-NLP/MLQuestions",
+        description=(
+            "MLQuestions is a domain adaptation dataset for the machine learning domain"
+            "It consists of ML questions along with passages from Wikipedia machine learning pages (https://en.wikipedia.org/wiki/Category:Machine_learning)"
+        ),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["dev", "test"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=("2021-01-01", "2021-03-31"),
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=["Article retrieval"],
```

Comment:
```suggestion
        task_subtypes=["Question Answering"],
```
I believe article retrieval typically refers to news article retrieval, but we might want to rename it to match that.